connector-email/src/main/java/com/redhat/cloud/notifications/connector/email/model/User.java

```diff
@@ -0,0 +1,162 @@
+package com.redhat.cloud.notifications.connector.email.model;
+
+import io.vertx.core.json.JsonObject;
+
+import java.util.Objects;
+
+/**
+ * Represents the internal user model used to effectively unify and process the
+ * users from the user providers.
+ */
+public final class User {
+
+    private String id;
+    private String username;
+    private String email;
+    private String firstName;
+    private String lastName;
+    private boolean isActive;
+    private boolean isAdmin;
+
+    public User() { }
+
+    public User(
+        String id,
+        String username,
+        String email,
+        String firstName,
+        String lastName,
+        boolean isActive,
+        boolean isAdmin
+    ) {
+        this.id = id;
+        this.username = username;
+        this.email = email;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.isActive = isActive;
+        this.isAdmin = isAdmin;
+    }
+
```

Comment:
This is never used.
```suggestion
```

Reply:
The `User` class has been removed, so this is not needed anymore.